doc_git.md

```diff
@@ -0,0 +1,90 @@
+# OUR FLOW :
+
+#### Branch creation:
+
+We always work on our own branch, which will be named in the same fashion : "[initials]/[feature]" == el/my_new_branch  
+To create such a branch  
+	`git checkout -b [initials]/[branch name]`
+
+/!\ No one is allowed to code on this branch, except its owner. If you need to test it or you need some of the code, you **must** create a new branch using the above command.  
+
+#### Branch rebase to apply changes after a Pull Request(PR):
+
+Make sure you are on master with:  
```

Comment:
another synonym for this command would be `git switch`.

Git added the command `git switch` to be more explicit whereas you are changing to a new branch or to another commit for a "detached head mode".